analysis/statistics/nr_routes.py

```diff
@@ -0,0 +1,10 @@
+from typing import Tuple
+
+from heuristic.classes import Solution
+
+
+def nr_routes(solution: Solution) -> Tuple[str, float]:
```

Comment:
We have a convention for the word number, it's not _nr_. It's _num_. Use _num_. Don't come up with alternative ways to write something that's already used throughout the code base.